Dockerfile

```diff
@@ -42,6 +42,7 @@ RUN dnf upgrade -y \
     chromium-$CHROMIUM_VERSION \
     firefox-$FIREFOX_VERSION \
     python2-pip-$PYTHON_PIP_VERSION \
```

Comment:
Is there a reason to keep Python 2? Can't we simply remove it to reduce the image size?

Reply:
Maybe I could, but it was less than a mb difference and this would let people revert back by using an alternative command. Also its installed in the base image so may be used by system components - would have to do some extra checking